src/Database/Adapter/MariaDB.php

```diff
@@ -1238,7 +1244,11 @@ public function sum(string $collection, string $attribute, array $queries = [],
 
         $stmt->execute();
 
-        $result = $stmt->fetch();
+        $result = $stmt->fetchall();
```

Comment:
Let's make sure we use proper casing:

```suggestion
        $result = $stmt->fetchAll();
```